eng/common/scripts/Helpers/ApiView-Helpers.ps1

```diff
@@ -62,4 +62,51 @@ function Check-ApiReviewStatus($packageName, $packageVersion, $language, $url, $
   {
     Write-Warning "Failed to check API review status for package $($PackageName). You can check http://aka.ms/azsdk/engsys/apireview/faq for more details on API Approval."
   }
+}
+
+function Process-ReviewStatusCode($statusCode, $packageName, $apiApprovalStat, $packageNameStat)
```

Comment:
I'd recommend the full name Status vs Stat on these variables. Stat might lead people to think about statistics instead.